Remotion/Validation/UnitTests/Validators/NotEmptyOrWhitespaceValidatorTest.cs

```diff
@@ -184,14 +184,14 @@ public void Validate_WithObject_ReturnsNoValidationFailures ()
       Assert.That(validationFailures, Is.Empty);
     }
 
-    private static void AssertValidationFailure (PropertyValidationFailure[] validationFailures, PropertyValidatorContext propertyValidatorContext)
+    private static void AssertValidationFailure (ValidationFailure[] validationFailures, PropertyValidatorContext propertyValidatorContext)
     {
       Assert.That(validationFailures.Length, Is.EqualTo(1));
       Assert.That(validationFailures[0].ErrorMessage, Is.EqualTo("The value must not be empty or contain only whitespace characters."));
       Assert.That(validationFailures[0].LocalizedValidationMessage, Is.EqualTo("Custom validation message."));
       Assert.That(validationFailures[0].ValidatedObject, Is.SameAs(propertyValidatorContext.Instance));
-      Assert.That(validationFailures[0].ValidatedProperty, Is.SameAs(propertyValidatorContext.Property));
-      Assert.That(validationFailures[0].ValidatedPropertyValue, Is.EqualTo(propertyValidatorContext.PropertyValue));
+      Assert.That(validationFailures[0].ValidatedProperties.Single().Property, Is.SameAs(propertyValidatorContext.Property));
+      Assert.That(validationFailures[0].ValidatedProperties.Single().ValidatedPropertyValue, Is.EqualTo(propertyValidatorContext.PropertyValue));
```

Comment:
You can write those with select expressions and list equality like in Remotion/Validation/UnitTests/Validators/NotEqualValidatorTest.cs etc